ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
     box-sizing: border-box;
     width: 400px;
```

Comment:
The background color has been changed from `#1e1e1e` to `#2b1d3c`. Could you provide more context on why this color change was made and if it aligns with the overall design theme?